src/lib.rs

```diff
@@ -175,8 +178,8 @@ impl Flatc {
     }
 
     /// New `flatc` command from specified path
-    pub fn from_path(path: PathBuf) -> Flatc {
-        Flatc { exec: path }
+    pub fn from_path(path: &Path) -> Flatc {
```

Comment:
Well, to avoid the API breaking change, we need to do this:

```suggestion
    pub fn from_path<P: std::convert::Into<PathBuf>>(path: P) -> Flatc {
```